src/components/ebay-date-textbox/index.marko

```diff
@@ -0,0 +1,49 @@
+// @ts-check
+
+<span.date-textbox
+    on-expander-expand("openPopover")
+    on-expander-collapse("closePopover")>
+    <if(input.range)>
+        <ebay-textbox
+            placeholder="YYYY-MM-DD"
```

Comment:
This needs to be customizable.

Reply:
I made it customizable, but the logic might not be ideal. Let me know what you think.